endpoints/openrtb2/amp_auction.go

```diff
@@ -809,3 +824,23 @@ func setTrace(req *openrtb2.BidRequest, value string) (err error) {
 
 	return nil
 }
+
+// setSeatNonBid populates bidresponse.ext.prebid.seatnonbid if  bidrequest.ext.prebid.returnallbidstatus is true
```

Comment:
Nitpick: could you remove extra space in between if and bidrequest in this comment